packages/cactus-plugin-cc-tx-visualization/.gitignore

```diff
@@ -0,0 +1,4 @@
+cactus-openapi-spec-plugin-consortium-manual.json
+src/main/typescript/generated/openapi/typescript-axios/.npmignore
+src/main/csv
```

Comment:
@RafaelAPB If we are ignoring `src/main/csv` then why is there a file committed in that directory? 
This question of mine might become irrelevant if you move it to the test directory - unless if that move is done then you'd also plan on updating the ignore pattern here.

Reply:
because that is a test file useful for people trying out the plugin. It was moved to the test directory now.

Reply:
> because that is a test file useful for people trying out the plugin. It was moved to the test directory now.

@RafaelAPB Gotcha, thank you. It is fine to have it in the test directory then.